.github/workflows/ttnn-stress-tests.yaml

```
@@ -0,0 +1,41 @@
+name: "ttnn stress tests"
+
+on:
+  workflow_dispatch:
+    inputs:
+      arch:
+        required: true
+        type: choice
+        options:
+          - wormhole_b0
+          - blackhole
+      runner-label:
+        required: true
+        type: choice
+        options:
+          - N150
+          - N300
+          - BH
```

To test specific blackhole cards I suggest the following:

```suggestion
          - P100
          - P150
```